### PR TITLE
Math worlds main page

### PR DESCRIPTION
**Pull request by CriticalWasTaken**

### CODE REVIEW

File: blog_page.css [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.text-box {
+    position: absolute;
+    text-align: center;
+    left: 50vw;
+    justify-content: center;
+}
```
Comment (dubinsky) [inline blog_page.css]: please add a newline character at the end of the file

File: blog_page.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+<body>
+    <p class = text-box>
+        Lorem Ipsum and some other words
+        <br>
+        need to check more lines
+        <br>
+        still working?
+    </p>
+</body>
+</html>
```
Comment (dubinsky) [inline blog_page.html]: newline

File: index.html [hunk 3]
```diff
@@ -170,19 +84,51 @@ <h3>MW Games</h3>
...
 	</footer>
+
+	<footer class = footer2>
+	<p>
+		MathWorlds is in the process of obtaining 501(c) nonprofit status.
+	</p>
+	</footer>
 </body>
-</html>
+</html>
```
Comment (dubinsky) [inline index.html]: newline

File: main.css [hunk 4]
```diff
@@ -1,70 +1,85 @@
...
 	padding: 6px 12px;
-	border: 1px solid #ccc;
+	border: 1px solid #a20000;
 	position: fixed;
-	float: right;
 	top: 150px;
-	padding: 1vw;
-}
+	background-color: bisque;
+}
```
Comment (dubinsky) [inline main.css]: newline

**Review by dubinsky — CHANGES_REQUESTED** [review]

**Comment by CriticalWasTaken** [comment]
New lines added, this code should be ready for deployment.

**Review by dubinsky — APPROVED** [review]

**Pull request merged**